File: python/java_to_python/06-pgmAvion/src/co/edu/campusucc/poo/TipoComercial.py
import time
from Avion import Avion  # assuming there is a class called Avion in avion.py

class TipoComercial(Avion):
    def __init__(self):
        super().__init__()
        self.setTipo("Comercial")

    def startingEngine(self):
        print("⛽Check Fuel ✅")
        for i in range(3):
            print("💥💥Spark💥💥")
            time.sleep(1 - (i * 0.2))
        print("Started the Motor...✈️✈️")

    def stopingEngine(self):
        print("✈️✈️ Check Engine ✅")
        for i in range(2):
            print("⛔ Stoping Engine...⛔‼️")
            time.sleep(1 - (i * 0.4))
        print("Stoped...⛔")

File: ruby/pgms_ruby/06-pgmAvion/TipoComercial.rb
require_relative 'Avion'
require 'time'

class TipoComercial < Avion
  def initialize
    super
    setTipo("Comercial")
  end

  def startingEngine
    puts("⛽Check Fuel ✅")
    3.times do |i|
      puts("💥💥Spark💥💥")
      sleep(1 - (i * 0.2))
    end
    puts("Started the Motor...✈️✈️")
  end

  def stopingEngine
    puts("✈️✈️ Check Engine ✅")
    2.times do |i|
      puts("⛔ Stoping Engine...⛔‼️")
      sleep(1 - (i * 0.4))
    end
    puts("Stoped...⛔")
  end
end